nvdaHelper/vbufBackends/gecko_ia2/gecko_ia2.cpp

```diff
@@ -552,14 +558,13 @@ VBufStorage_fieldNode_t* GeckoVBufBackend_t::fillVBuf(IAccessible2* pacc,
 		|| role == ROLE_SYSTEM_BUTTONMENU
 		|| ((role == ROLE_SYSTEM_CHECKBUTTON || role == ROLE_SYSTEM_RADIOBUTTON) && !isLabelVisible(pacc));
 	// Whether this node has a visible label somewhere else in the tree
-	const bool labelVisible = canDetectLabelVisibility // Not all browsers support getting a node's labelledBy node
-		&& nameIsExplicit && name && name[0] //this node must actually have an explicit name, and not be just an empty string
+	const bool labelVisible = nameIsExplicit && name && name[0] //this node must actually have an explicit name, and not be just an empty string

```

Comment:
Do we care what character `name[0]` is? If it is required to be visible, do we need to test that it is not a "non-printable" character?

Reply:
I think this question is still valid, even though github has hidden it.

Reply:
I believe what we have is correct, as  if nameIsExplicit, then the author has provided some kind of accessible label, and if it contains anything (even a space) clearly they are wanting to inforce the content, E.g. overriding existing descendant content with a space.